.storybook/stories/SelectableTableBatchActions/ModalAddTrackingNumber.tsx

```diff
@@ -31,14 +34,15 @@ export function ModalAddTrackingNumber({
 	return (
 		<Drawer
 			isOpen={isOpen}
+			onClose={onClose}
 			onDismiss={onDismiss}
```

Comment:
We shouldn't be using `onDismiss` any more

Reply:
Removed